app/components/channel_actions/channel_actions.tsx

```diff
@@ -13,15 +14,14 @@ import {useServerUrl} from '@context/server';
 import {dismissBottomSheet} from '@screens/navigation';
 import {isTypeDMorGM} from '@utils/channel';
 
-// import AddPeopleBox from '@components/channel_actions/add_people_box';
```

Comment:
should this be deleted?

Reply:
Yes. The reason is it is no longer called `AddPeopleBox` but `AddMembersBox`.